src/style/AccountPage.less

```diff
@@ -15,12 +15,101 @@
 	}
 }
 
-#account {
-	.navbar {
-		.login-link {
-			color: #AD2016;
-			& a {
-				color: #AD2016;
+.AccountPage {
+
+	.user {
+		img {
+			width: 100%;
+			@media @md-max {
+				width: 25%;
+				margin-bottom: 1rem;
+			}
+		}
+		h1 {
+			font-size: 1.6rem;
+			font-weight: 600;
+		}
+	}
+
+	h1, h2, h3, h4 {
```

Comment:
As above -- Can we remove this? If not, why? Thanks

NB: this comment applies for every font edit really, but I won't repeat it as that'd be tiresome.